### PR TITLE
Added a storage utility

### PR DESCRIPTION
**Pull request by WebReflection**
This *MR* exposes yet another handy utility behind the `storage` name.

It allows users to initiate a persistent IndexedDB based *Map* that will act synchronously after initialization.

/cc @ntoll

### CODE REVIEW

File: test/integration/interpreter/micropython/storage.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <script type="module">
+        import { init } from '../utils.js';
+        init('micropython');
+    </script>
+</head>
+<body>
+  <script type="micropython" async>
+    from polyscript import storage
+
+    map = await storage("@polyscript/storage")
```
Comment (ntoll) [inline test/integration/interpreter/micropython/storage.html:15]: Perhaps rename `map` to `store`..? To my ears it sounds more natural. Feel free to ignore.

**Review by ntoll — APPROVED** [review]
:shipit:

File: test/integration/interpreter/micropython/storage.html (hunk 1 above)
Comment (WebReflection) [reply test/integration/interpreter/micropython/storage.html:15]: it's just an example ... you can name it *s* if you prefer ... I don't mind, though it's literally an `instanceof Map`

Comment (WebReflection) [reply test/integration/interpreter/micropython/storage.html]: done

**Pull request merged**